src/pages/home/Home.jsx

```diff
@@ -1,9 +1,104 @@
-import React from 'react'
-
+import {React, useEffect, useState} from 'react';
+import axios from 'axios';
+import BackgroundIamge from "../../assets/background-image.png";
+import "../../../src/App.css";
+import {Link} from "react-router-dom";
 function Home() {
+    const [category, setCategory] = useState([]);
+
+    useEffect(() => {
+        (async () => {
+            try {
+                const response = await axios.get('/api/categories');
```

Comment:
I would suggest defining api endpoints in a single file and export them from that file, so that any changes to the endpoints can be reflected everywhere without editing the entire code.

Reply:
ok 👍🏻